### ISSUE TITLE
ValueError on 'sanitize' True of type <class 'numpy.bool_'> (util.py)

### ISSUE THREAD

**Issue by 12seetharaman** [allenai/allennlp#3287]
**Description**
"numpy.bool_" type was not included on Sanitize to convert it into python type. 
```
x = True   ## type(x) ->  numpy.bool_ 

    def sanitize(x: Any) -> Any:  # pylint: disable=invalid-name,too-many-return-statements
        """
        Sanitize turns PyTorch and Numpy types into basic Python types so they
        can be serialized into JSON.
        """
        if isinstance(x, (str, float, int, bool)):
            # x is already serializable
            return x
        elif isinstance(x, torch.Tensor):
            # tensor needs to be converted to a list (and moved to cpu if necessary)
            return x.cpu().tolist()
        elif isinstance(x, numpy.ndarray):
            # array needs to be converted to a list
            return x.tolist()
        elif isinstance(x, numpy.number):  # pylint: disable=no-member
            # NumPy numbers need to be converted to Python numbers
            return x.item()
        elif isinstance(x, dict):
            # Dicts need their values sanitized
            return {key: sanitize(value) for key, value in x.items()}
        elif isinstance(x, (spacy.tokens.Token, allennlp.data.Token)):
            # Tokens get sanitized to just their text.
            return x.text
        elif isinstance(x, (list, tuple)):
            # Lists and Tuples need their values sanitized
            return [sanitize(x_i) for x_i in x]
        elif x is None:
            return "None"
        elif hasattr(x, 'to_json'):
            return x.to_json()
        else:
>           raise ValueError(f"Cannot sanitize {x} of type {type(x)}. "
                             "If this is your own custom class, add a `to_json(self)` method "
                             "that returns a JSON-like object.")
E           ValueError: Cannot sanitize True of type <class 'numpy.bool_'>. If this is your own custom class, add a `to_json(self)` method that returns a JSON-like object.
```

**Call log**
```
env/lib/python3.7/site-packages/allennlp/predictors/predictor.py:213: in predict_batch_instance
    return sanitize(outputs)
env/lib/python3.7/site-packages/allennlp/common/util.py:73: in sanitize
    return [sanitize(x_i) for x_i in x]
env/lib/python3.7/site-packages/allennlp/common/util.py:73: in <listcomp>
    return [sanitize(x_i) for x_i in x]
env/lib/python3.7/site-packages/allennlp/common/util.py:67: in sanitize
    return {key: sanitize(value) for key, value in x.items()}
env/lib/python3.7/site-packages/allennlp/common/util.py:67: in <dictcomp>
    return {key: sanitize(value) for key, value in x.items()}
env/lib/python3.7/site-packages/allennlp/common/util.py:67: in sanitize
    return {key: sanitize(value) for key, value in x.items()}
env/lib/python3.7/site-packages/allennlp/common/util.py:67: in <dictcomp>
    return {key: sanitize(value) for key, value in x.items()}
```
**To Reproduce**
Call sanitize function by passing dictionary contains 'numpy.bool_' type value in it.

**Expected behavior**
Need to convert 'numpy.bool_' to 'bool'

**System (please complete the following information):**
 - OS: Ubuntu
 - Python version: 3.7
 - AllenNLP version: 0.9.0

**Comment by joelgrus** [comment]
seems like a reasonable thing to want, PR welcome

**Comment by joelgrus** [comment]
fixed by #3290
